connman/plugins/sailfish_ofono.c

```diff
@@ -100,8 +100,12 @@ enum connctx_handler_id {
 #define ONLINE_CHECK_SEC  (2)
 /* Timeout for delayed set connected call, in milliseconds. */
 #define DELAYED_CONNECT_TIMEOUT_MS (100)
-/* Wait for approx 2s for address information from ofono. */
-#define DELAYED_CONNECT_LIMIT (2000 / DELAYED_CONNECT_TIMEOUT_MS)
+
+/*
+ * By default wait for approx 5s (500*DELAYED_CONNECT_TIMEOUT_MS) for address
+ * information from ofono.
+ */
+static unsigned int delayed_connect_limit = 500;
```

Comment:
If the previous value `(2000 / DELAYED_CONNECT_TIMEOUT_MS) = (2000 / 100) = 20` was 2 seconds, it looks like 5 seconds should be 50.

Reply:
Ah yes, I did not remember this changes the interval as well.. Hm, maybe that should be static interval anyways.

Reply:
Argh, cannot remember my own changes. The DELAYED_CONNECT_LIMIT is the amount of cycles with the pre-defined DELAYED_CONNECT_TIMEOUT_MS wait intervals. So yes, the comment you had is quite correct but the context is that:
- old = 20 cycles with 100ms wait resulting in total of 2s wait
- new = 50 cycles with 100ms wait resulting in total of 5s wait